dbms/src/Flash/CoprocessorHandler.h

```diff
@@ -37,7 +37,8 @@ struct CoprocessorContext
     const kvrpcpb::Context & kv_context;
     const grpc::ServerContext & grpc_server_context;
 
-    CoprocessorContext(Context & db_context_, const kvrpcpb::Context & kv_context_, const grpc::ServerContext & grpc_server_context_);
+    CoprocessorContext(
+        Context & db_context_, const kvrpcpb::Context & kv_context_, const grpc::ServerContext & grpc_server_context_);
```

Comment:
Is it caused by `ColumnLimit`? This method declares all params in the following line.
I prefer the `CoprocessorHandler` style, which declares params in multiple lines.

I prefer:
* For the short one, the function name and its params are in the same line
* For the long one, the function name and all its params are in different lines

Reply:
Got it. I change the `AllowAllArgumentsOnNextLine` and `AllowAllParametersOfDeclarationOnNextLine` back to false.